elements/rh-alert/rh-alert.ts

```diff
@@ -43,8 +43,6 @@ export class AlertCloseEvent extends ComposedEvent {
 /**
  * An alert to display information on a website.
  *
- * By default it is black text on a black background
```

Comment:
@brianferry, I removed this from the JSdoc since that's not the default, but I want to make sure that's ok!

Reply:
Yup that looks good